index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link
+    rel="stylesheet"
+    href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+  />
+    <title>Awesome Books Library</title>
+    <link rel="stylesheet" href="index.css">
+</head>
+<body>
+    <header>
+        <nav class="nav-links">
+            <i class="fa-solid fa-arrow-left"></i>
+            <i class="fa-solid fa-arrow-right"></i>
+            <i class="fa-solid fa-arrow-down"></i>
+            <input type="search" placeholder="https://">
+            <button type="button"><i class="fa-solid fa-magnifying-glass"></i></button>
+        </nav>
```

Comment:
- [ ] [OPTIONAL] This section of your styling isn't necessary. It just represents the wireframe of a regular browser. If this section doesn't have any functionality on your site please remove it 🙏 

![image](https://user-images.githubusercontent.com/74814780/178597944-fe05aaf9-fe89-4880-a339-17cda313d65a.png)